src/react-components/input/Button.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import PropTypes from "prop-types";
+import classNames from "classnames";
+import styles from "./Button.scss";
+
+export function Button({ primary, secondary, cta, children, ...rest }) {
```

Comment:
Not a super strong opinion here but might be nice to instead have something like `type="primary"`, since they are mutually exclusive and then proptypes could enforce you are not making invalid states.

Reply:
My reasoning behind this was that `primary` and `cta` were being used together. Also `type` should be reserved for `type="submit"` etc.